### DISCUSSION TITLE
Proposal for Reimbursement of Dubai Trip Pending Expenses

### DISCUSSION THREAD

**Discussion by nblogist** [ringdao-community-guild/discussion#199] [Proposals Temperature Check]
**Author:** Furqan Ahmed  
**Date:** 1st Dec 2024

---

## Summary  
This proposal seeks DAO approval for the reimbursement of pending travel expenses related to the Dubai trip for Cosmoverse 2024. The trip was pivotal in strengthening Darwinia’s presence in the blockchain ecosystem, building valuable partnerships, and demonstrating our ongoing commitment to innovation and collaboration.

To ensure fairness, I have personally funded the initial leg of the journey (from the US to Singapore and Karachi), and I am only requesting reimbursement for the return flight to the US (Karachi → Salt Lake City), which directly supports my participation in Darwinia-related activities.

---

## Reimbursement Request Details  
1. **Return Flight (Karachi → Salt Lake City):** $1,000 (usually/approx)  
2. **Deficit from Dubai Trip Expenses:** $139.85  

**Total Amount Requested:** $1,139.85  

---

### 1. Cost-Sharing Approach  
The overall trip included both professional commitments to Darwinia and personal components. To ensure that DAO resources are used responsibly, I funded the outbound travel (US → Singapore → Karachi) independently. By only requesting reimbursement for the return flight (Karachi/Dubai → Salt Lake City), I am proposing a fair cost-sharing model that aligns Darwinia’s contributions with the professional aspects of the trip.  

### 2. Value Delivered to Darwinia  
- Established partnerships with key figures and organizations across ecosystems like Cardano, Osmosis, and Cointelegraph.  
- Participated in Cosmoverse 2024 as a speaker, increasing Darwinia’s visibility and credibility in the blockchain space.  
- Achieved significant _**savings**_ through strategic budgeting, including **_$1,464 SAVED_** in direct cost reductions for accommodation, conference fees, and other expenses.  

### 3. Transparency  
- A detailed breakdown of expenses and the trip’s outcomes is available in the [Dubai Report](https://docs.google.com/document/d/1DwL7geFF2JdUWbSkrNTXXDTiARyOLqZDd3pN6F5prMo/edit?tab=t.0). This ensures full accountability for the funds being requested.  
- Original Tally Proposal for Dubai [here](https://www.tally.xyz/gov/dcdao/proposal/50559279770185093185630632312686269587339353772333996266645150746493617082787)
- I am currently in Karachi.
---

## Potential Concerns Addressed  

- **Travel Path:** The proximity between Dubai and Karachi (1.5 hours) enabled efficient travel planning. My independent funding of the outbound leg ensures that Darwinia’s resources are applied only to the return flight directly linked to the project.  
- **Fairness:** By shouldering the costs of travel to Singapore and Karachi, I demonstrated a commitment to sharing expenses, ensuring DAO funds are used exclusively for Darwinia’s benefit.  
- **Impact:** The trip was a high-value opportunity for Darwinia, creating long-term partnerships and elevating the project’s presence in global blockchain ecosystems.  

---

Your approval of this reimbursement will affirm Darwinia’s commitment to supporting meaningful ecosystem engagements, ensuring the network continues to grow its influence and relevance in the blockchain industry. Thank you for considering this proposal.

**Comment by hackfisher** [comment]
Just as I didn’t support the original proposal, I’m not positive about this one either.

The reason I didn’t support the original proposal was that it mixed personal travel with attendance at Darwinia events, especially involving long-haul international trips. This inherently created complexity at the execution level, making it difficult to distinguish costs accurately. However, the original proposal had no procedural issues, as it was passed through proper governance processes, and its key data (e.g., sponsorship amounts and attended events) were unambiguous.

Additionally, it’s important to consider that the original proposal was approved based on its original budget and sponsorship goals. Two questions:

1. Will original proposal would still have been approved if the budget had increased?
2. Will we would approve it regardless of how high the costs were?

As for the current proposal, I do not support it, primarily because it conflicts with the original proposal (in terms of amounts) and its calculations are difficult to distinguish due to the mixing of personal travel. I will oppose the original proposal more strongly if its budget were to incorporate the amounts in this proposal.

**Reply by nblogist** [reply]
Thank you for your feedback - Due to the mixing of personal travel - that's why i proposed one way coverage only! But the questions do make it clear - i say we take this as our guide for future budget requirements and reporting standards. Happy to contribute as always!

**Comment by Laki389** [comment]
Hi @nblogist , wasnt this already voted 4 -2 in Tally governance and proposal was executed?

**Reply by Laki389** [reply]
> Hi @nblogist , wasnt this already voted 4 -2 in Tally governance and proposal was executed?

https://www.tally.xyz/gov/dcdao/proposal/50559279770185093185630632312686269587339353772333996266645150746493617082787

**Comment by Laki389** [comment]
oh ok... I realized it is about additional amount..
I think it would probably be good if you point that out in this temp. check because at least I didn't realize it until I read Denny's comment. :+1: :pray:

**Comment by Laki389** [comment]
![Screenshot_1](https://github.com/user-attachments/assets/cfd2d433-069f-4b16-a39d-05870d5c48d8)

Reading the already voted Tally proposal, Karachi is not mentioned while the return flight (Dubai → Salt Lake City, US) was already voted on.

I agree with @hackfisher  rhetorical question: Will the original proposal still have been approved if the budget was $2,840?

**Reply by nblogist** [reply]
Karachi- is 1-1.5 hrs away from Dubai - flight costs almost the same out of Dubai/Karachi - in Karachi rn for logistical reasons.

**Comment by nblogist** [comment]
Thank you so much for writing @Laki389 @hackfisher 

Since it is clear that this wouldn't have approved - given the rhetorical question - I am happy to chalk this as my contribution to the DCDAO; while I urge that we use this instant as the north star in future conference proposals in terms of expectations- reporting, and transparency guidelines and ofc budgeting.
